pkgs/applications/version-management/git-and-tools/git-when-merged/default.nix

```diff
@@ -0,0 +1,39 @@
+{ stdenv, fetchFromGitHub, python3 }:
+
+stdenv.mkDerivation rec {
+  pname = "git-when-merged";
+  version = "1.2.0";
+
+  src = fetchFromGitHub {
+    owner = "mhagger";
+    repo = pname;
+    rev = version;
```

Comment:
```suggestion
    rev = "v${version}";
```

Reply:
arghhhh. It's not the first time I get bitten by that: I make changes and the new package still works. I just forgot that the src SHA hasn't change and that I can introduce as many bugs as I want :-(.

Thank you so much for catching that. How is it possible that the CI didn't detect the bug either? Is it because it uses the same /nix/store to build the PR several times?

Reply:
>Is it because it uses the same /nix/store to build the PR several times?

yes, that is what probably happened. Ofborg used the same runner for the build, hence reusing the cached output